src/components/tl/mlx5/tl_mlx5.c

```diff
@@ -104,6 +104,15 @@ static ucc_config_field_t ucc_tl_mlx5_lib_config_table[] = {
      ucc_offsetof(ucc_tl_mlx5_lib_config_t, mcast_conf.one_sided_reliability_enable),
      UCC_CONFIG_TYPE_BOOL},
 
+    {"MCAST_ZERO_COPY_ALLGATHER_ENABLE", "1", "Enable truly zero copy allgather design for mcast",
```

Comment:
what does "truly" mean in this context?

Reply:
we already had a zero-copy design where there is no copy on the sender side. However, in that design, there is still a copy required on the receiver side. The new proposed design is completely zero-copy therefore, it is called Truly zero-copy.

Reply:
ok, I leave it up to you to change the comments but IMO that's not straightforward for the reader